features/step_definitions/file_steps.coffee

```diff
@@ -5,17 +5,21 @@ path = require 'path'
 module.exports = ->
 
   @Given /^I have no test files$/, (done) ->
-    done() # No op for readability
+    done()
 
 
-  @Given /^I have 1 (passing|failing) test$/, (status, done) ->
-    outputTestFile path.join(@tmpDir, 'example_spec.coffee'), passing: status is 'passing', done
+  @Given /^I have a (passing|failing) test$/, (status, done) ->
+    outputTestFile path.join(@tmpDir, 'example_spec.coffee'),
+                   passing: status is 'passing',
+                   done
 
 
-  @Given /^I have the file "([^"]*)" with 1 (passing|failing) test$/, (filePath, status, done) ->
-    outputTestFile path.join(@tmpDir, filePath), passing: status is 'passing', done
+  @Given /^I have a file "([^"]*)" containing a (passing|failing) test$/, (filePath, status, done) ->
+    outputTestFile path.join(@tmpDir, filePath),
+                   passing: status is 'passing',
+                   done
 
 
-  @Then /^I now have the file "([^"]*)" with the contents$/, (fileName, fileContent, done) ->
+  @Then /^my code base now has a file "([^"]*)" containing$/, (fileName, fileContent, done) ->
```

Comment:
In my opinion we need to stop doing that. I think that's mostly an unconscious application of a pattern, over and over. We should use "I" only if we describe a user action.